src/main/logic/Logic.cpp

```diff
@@ -345,6 +357,8 @@ void Logic::startNewGame(){
 	eventManager->fireEvent(gameStateChangedEvent);
 	//Now we need to initalise everything
 	generateLevel();
+	//Setup the players
+	numLives = 1;
```

Comment:
I'd make sure this is set in the constructor, otherwise it'll be a random value from memory and might produce some weird effects if we forget later on.